### ISSUE TITLE
ApolloClient throwing error when used with meteorClientConfig()

### ISSUE THREAD

**Issue by purplecones** [apollostack/apollo-client#346]
I get this error when passing meteorClientConfig() into ApolloClient(). If I remove meteorClientConfig(), everything works fine 

![localhost_3000_account_investment-accounts](https://cloud.githubusercontent.com/assets/136654/16523896/bcdea07e-3f73-11e6-9508-765371a74d92.png)

configure client
```
const Client = new ApolloClient(meteorClientConfig());
const logger = createLogger();
const middlewares = [
  Client.middleware(),
];
```
```
const reducers = {
  apollo: Client.reducer(),
};
```
```
const Store = createStore(
  combineReducers(reducers),
  compose(
    applyMiddleware(...middlewares),
    window.devToolsExtension ? window.devToolsExtension() : undefined,
  )
);
```

configure server
```
function configureGraphQLServer(options = {}) {
  /* eslint-disable global-require*/
  const express = require('express');
  const apolloServer = require('apollo-server').apolloServer;
  const proxyMiddleware = require('http-proxy-middleware');

  const Meteor = Package['meteor'].Meteor;
  const Accounts = Package['accounts-base'].Accounts;
  const WebApp = Package['webapp'].WebApp;
  const check = Package['check'].check;

  if (!Meteor) {
    const error = 'Meteor package is missing';
    throw new Error(error);
  }

  if (!Accounts) {
    const error = 'accounts-base package is missing';
    throw new Error(error);
  }

  if (!WebApp) {
    const error = 'Webapp package is missing';
    throw new Error(error);
  }

  if (!check) {
    const error = 'check package is missing';
    throw new Error(error);
  }

  const {
    schema,
    resolvers,
    port = 4000,
    urlName = 'graphql',
    graphiql = true,
    pretty = true,
    context = {},
    ...others,
  } = options;

  const graphQLServer = express();
  const GRAPHQL_PORT = port;

  graphQLServer.use(`/${urlName}`, apolloServer(async (req) => {
    let userId = null;

    /* eslint-disable no-underscore-dangle */
    /* eslint-disable no-param-reassign */
    if (req.headers.meteorlogintoken) {
      const token = req.headers.meteorlogintoken;
      console.log('token', token);
      check(token, String);
      const hashedToken = Accounts._hashLoginToken(token);

      // Get the user from the database
      const user = await Meteor.users.findOne({
        'services.resume.loginTokens.hashedToken': hashedToken,
      }, {
        fields: {
          _id: 1,
          'services.resume.loginTokens.$': 1,
        },
      });

      if (user) {
        const expiresAt = user.services.resume.loginTokens[0].when;
        const isExpired = expiresAt < Date.now(); // TODO or new Date()

        if (!isExpired) {
          userId = user._id;
        }
      }
    }

    return {
      graphiql,
      pretty,
      schema,
      resolvers,
      context: {
        ...context,
        userId,
      },
      ...others,
    };
  }));

  graphQLServer.listen(GRAPHQL_PORT, () => console.log(`
    GraphQL Server is now running on http://localhost:${GRAPHQL_PORT}
    Checkout http://localhost:${GRAPHQL_PORT}/${urlName} to use GraphiQL
  `));

  WebApp.rawConnectHandlers.use(proxyMiddleware(`http://localhost:${GRAPHQL_PORT}/graphql`));
}

export default configureGraphQLServer;
```

**Comment by abhiaiyer91** [comment]
What is meteorClientConfig?

**Comment by purplecones** [comment]
Comes from the meteor package `apollo` : 
http://docs.apollostack.com/apollo-client/meteor.html#meteorClientConfig

**Comment by stubailo** [comment]
Still have to update meteor starter kit etc

**Comment by stubailo** [comment]
@purplecones can you try updating all of your dependencies and let me know if this still happens?

**Comment by purplecones** [comment]
@stubailo I still get this error with `ApolloClient(meteorClientConfig())`. Calling `ApolloClient()` on its own is working fine.

![localhost_3000_account_investment-accounts](https://cloud.githubusercontent.com/assets/136654/16590391/5c7748c8-42a5-11e6-90b7-073e38c85490.png)

Updated to to `0.3.26`:
```
"dependencies": {
    "alasql": "^0.2.6",
    "apollo-client": "^0.3.26",
    "apollo-server": "^0.1.5",
    "apollo-tools": "0.0.20",
    "classnames": "^2.2.5",
    "express": "^4.14.0",
    "fingerprintjs2": "^1.1.4",
    "graphql": "^0.6.0",
    "graphql-tag": "^0.1.7",
    "http-proxy-middleware": "^0.16.0",
    "immutable": "^3.8.1",
    "load-script": "^1.0.0",
    "lodash": "^4.12.0",
    "mantra-apollo": "^1.0.2",
    "mantra-core": "^1.6.1",
    "mantra-plus": "^2.0.7",
    "mantra-redux": "^1.0.5",
    "material-ui": "^0.15.0",
    "meteor-auth": "^0.1.1",
    "meteor-node-stubs": "^0.2.3",
    "moment": "^2.13.0",
    "numeral": "^1.5.3",
    "pg": "^6.0.1",
    "pg-hstore": "^2.3.2",
    "react": "^15.1.0",
    "react-dom": "^15.0.2",
    "react-komposer": "^1.8.0",
    "react-komposer-plus": "^2.0.8",
    "react-komposer-redux": "0.0.3",
    "react-mounter": "^1.2.0",
    "react-redux": "^4.4.5",
    "react-semantify": "^0.5.1",
    "react-tap-event-plugin": "^1.0.0",
    "redux": "^3.5.2",
    "redux-form": "^5.2.3",
    "redux-logger": "^2.6.1",
    "redux-thunk": "^2.1.0",
    "sequelize": "^3.23.4",
    "simple-react-form": "^1.2.0",
    "simple-react-form-material-ui": "^1.2.2",
    "synapsepay": "0.0.3",
    "underscore.string": "^3.3.4",
    "url-parse": "^1.1.1"
  },
```

**Comment by helfer** [comment]
@purplecones: This error is more pertinent to the meteor integration package, so you should file it there. I've seen this kind of error before, and it's due to the fact that Node.js 0.12 doesn't have a split method on object. You should be able to fix it by updating to Meteor 1.4.
